tests/ut/py/test_worker/test_admission_fence.py

```
@@ -0,0 +1,289 @@
+@pytest.fixture
+def ready_worker():
+    """A READY L3 worker with one SUB child and no chips — no NPU required."""
+    w = Worker(level=3, num_sub_workers=1, startup_timeout_s=_TEST_WALL_BUDGET_S)
+    w.init()
+    try:
+        yield w
+    finally:
+        try:
+            w.close()
+        except BaseException:  # noqa: BLE001 -- a test may have closed it already
+            pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)